ethereum/retesteth
# version 0.1.1 development

* fix --checkhash warnings on LegacyTests
* remove blockchain version of old VMTests
* typed transactions exported without rlp prefix (as plain `01 + tx.bytes`)
* select transaction by label in a filled state test
* check for excessive expect sections in gstate tests
* yul compilation support
* t8ntool secret key export
* transaction type2 support in gstate tests (0x02)
* DataObject notinitialized type for std logic. fix issues with autosort.
* openethereum-evm gstate tests support via wrapper
* replace BlockHeader with spBlockHeader
* introduce new struct BlockHeader1559
* support besu and oewrap
* EIP1559 test parsing
* setup.sh script in client configs. (executed once in the begining of retesteth run)
* fieldReplace map in clients config (on rpc protocol)
* checkLogHash option in clients config
* check t8ntool response files for emptiness bugfix
* eip1559 blockchain test support (t8ntool)
* eip1559 BerlinToLondonAt5 support (t8ntool)
* invalid transactions check
* test_mineBlocks method expect result of rejected transactions
